src/seedu/addressbook/data/person/Person.java

```diff
@@ -39,6 +39,17 @@ public Name getName() {
         return name;
     }
 
+    public int compareTo(Person p2) {
+        if (this.name.toString().compareTo( p2.getName().toString())> 0) {
```

Comment:
In this method, `return this.name.toString().compareTo( p2.getName().toString());`
will be enough and it is more concise. Do you agree with that?